packages/iov-keycontrol/src/wallets/slip10wallet.ts

```diff
@@ -171,12 +167,16 @@ export class Slip10Wallet implements Wallet {
   public readonly implementationId = "override me!" as WalletImplementationIdString;
   public readonly id: WalletId;
 
+  // wallet
   private readonly secret: EnglishMnemonic;
   private readonly curve: Slip10Curve;
-  private readonly identities: LocalIdentity[];
-  private readonly privkeyPaths: Map<string, ReadonlyArray<Slip10RawIndex>>;
   private readonly labelProducer: DefaultValueProducer<string | undefined>;
 
+  // identities
+  private readonly identities: PublicIdentity[];
+  private readonly privkeyPaths: Map<IdentityId, ReadonlyArray<Slip10RawIndex>>;
+  private readonly labels: Map<IdentityId, string | undefined>;
```

Comment:
This repeats the Map mostly...
Maybe something like 
```ts
private readonly idData: Map<IdentityId, IdentityData>

interface IdentityData {
  readonly paths: ReadonlyArray<Slip10RawIndex>;
  readonly label: string | undefined;
}
```

I think this will make it easier to add more in the future.
But maybe not worth the change... fine as it is, you can take or leave the suggestion

Reply:
Good idea. Will update as soon as we have 3 properties per IdentityId